package.json

```diff
@@ -1,41 +1,41 @@
 {
-  "name": "runit",
-  "private": true,
-  "version": "1.0.0",
-  "description": "online editor for coding",
-  "engines": {
-    "node": ">=18.13.0"
-  },
-  "workspaces": [
-    "backend",
-    "frontend"
-  ],
-  "scripts": {
-    "build:backend": "yarn workspace backend run build",
-    "build:frontend": "DISABLE_ESLINT_PLUGIN=true yarn workspace frontend run build",
-    "build": "npm-run-all --parallel build:*",
-    "production": "yarn workspace backend run start:prod",
-    "start:backend": "yarn workspace backend run start:debug",
-    "start:frontend": "yarn workspace frontend run start",
-    "start": "npm-run-all --parallel start:*",
-    "test": "playwright test --trace on" 
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/hexlet-rus/runit.git"
-  },
-  "author": "",
-  "license": "Affero GPL 3.0",
-  "bugs": {
-    "url": "https://github.com/hexlet-rus/runit/issues"
-  },
-  "homepage": "https://github.com/hexlet-rus/runit#readme",
-  "dependencies": {
-    "npm-run-all": "^4.1.5"
-  },
-  "devDependencies": {
-    "@nestjs/cli": "^9.2.0",
-    "@playwright/test": "^1.30.0",
-    "node-gyp": "^9.3.1"
-  }
+	"name": "runit",
+	"private": true,
+	"version": "1.0.0",
+	"description": "online editor for coding",
+	"engines": {
+		"node": ">=18.13.0"
+	},
+	"workspaces": [
+		"backend",
+		"frontend"
+	],
+	"scripts": {
+		"build:backend": "yarn workspace backend run build",
+		"build:frontend": "DISABLE_ESLINT_PLUGIN=true yarn workspace frontend run build",
+		"build": "npm-run-all --parallel build:*",
+		"production": "yarn workspace backend run start:prod",
+		"start:backend": "yarn workspace backend run start:debug",
+		"start:frontend": "yarn workspace frontend run start",
+		"start": "npm-run-all --parallel start:*",
+		"test": "playwright test --trace on"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/hexlet-rus/runit.git"
+	},
+	"author": "",
+	"license": "Affero GPL 3.0",
+	"bugs": {
+		"url": "https://github.com/hexlet-rus/runit/issues"
+	},
+	"homepage": "https://github.com/hexlet-rus/runit#readme",
+	"dependencies": {
+		"npm-run-all": "^4.1.5"
+	},
+	"devDependencies": {
+		"@nestjs/cli": "^9.2.0",
+		"@playwright/test": "^1.30.0",
+		"node-gyp": "^9.3.1"
+	}
```

Comment:
@SidorovVladimir can you undo changes here and fix editors settinns? By default in package.json file indent is 2 spacees